server/repository/userRepository.js

```diff
@@ -170,6 +112,24 @@ class UserRepository {
     });
     return allUsers || null;
   }
+
+  /**
+   * Change confirmEmail field to true
+   * @param {integer} userId User Id
+   * @returns {object} i dont know yet
+   */
+  static async confirmUserEmail(userId) {
+    const user = await User.findByPk(userId);
```

Comment:
Nice 👍

Reply:
but what happens if the user is not found?

Reply:
@seyi-adeleke, wow, I am going to fix this ASAP. Thanks for pointing it out.